tests/serverpod_test_server/test/defaults/exception/enum_default_test.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  group(
+    "Given an exception with 'default' fields",
+    () {
+      test(
+        'when an exception is created, then the "defaultMessage" field should match the default value',
+        () {
+          var exception = DefaultException();
```

Comment:
Suggestion: Validated all data types.

When you expand the model, add check that this works for all the data types as well.